salt/states/pcs.py

```diff
@@ -508,23 +526,39 @@ def cluster_setup(name, nodes, pcsclustername="pcscluster", extra_args=None):
             value = line.split(":")[1].strip()
             if key in ["Cluster Name"]:
                 if value in [pcsclustername]:
-                    ret["comment"] += "Cluster {0} is already set up\n".format(
+                    ret["comment"] += "Cluster {} is already set up\n".format(
                         pcsclustername
                     )
                 else:
                     setup_required = True
                     if __opts__["test"]:
-                        ret["comment"] += "Cluster {0} is set to set up\n".format(
+                        ret["comment"] += "Cluster {} is set to set up\n".format(
                             pcsclustername
                         )
 
     if not setup_required:
+        log.info("No setup required")
         return ret
 
     if __opts__["test"]:
         ret["result"] = None
         return ret
 
+    # Debian based distros deploy corosync with some initial cluster setup. 
+    # The following detects if it's a Debian based distro and then stops Corosync
+    # and removes the config files. I've put this here because trying to do all this in the
+    # state file can break running clusters and can also take quite a long time to debug.
+    
+
+    log.debug("OS_Family: %s", __grains__.get("os_family"))
```

Comment:
I'm hesitant about this change. What if a user doesn't  want to remove these files and it removes files they intend to be there?

Reply:
If the files exist and are consistent with the required state then it won't get to the file deletion. The deletion only runs when there is a cluster with the incorrect name installed, which is the initial state for Debian and Ubuntu. The only time this can be destructive is when there is a cluster installed that doesn't match the required clusters name, e.g., the incorrect formula is applied to the incorrect host. I think it is worth the risk. If you disagree I can try target that default cluster a bit better but deleting that default cluster is definitely worth doing.

Reply:
ping @garethgreenaway could I get your opinion here ^

Reply:
I share the hesitancy here as well.  I understand the rational for the change I'm weary of making these kind of changes.  Anytime something is being done in a deterministic manner outside of Salt without input from the user it's a potential for problems.  I wonder if a better approach would be to detect if the default cluster exists and show a warning suggesting to the user that they delete it?  Part of the warning could be particular salt commands to run to clear out the default databases.

Reply:
That will make it pretty much impossible to run this automatically which will really introduce some frustration. They really are placeholders and don't exist on other systems. Maybe the middle ground is to check for a the default cluster name being debian and then deleting it? Or adding some sort of "wipe if default" flag that can allow for removing the temporary cluster.

Reply:
Completely understand, I would just hate to see someone whose actively using the default cluster have something deleted without their knowledge that it's going to be deleted.  I like the idea of having something like a `wipe_default` flag that defaults to False.